open-budget-il/upload.py

```diff
@@ -91,30 +91,49 @@ def handle(self, response, item, current_doc):
 
         doc = None
         # Update the document if the dirty_fields contain any of the FTS_FIELDS
-        fts_field_names = map(lambda x: x[0], self.FTS_FIELDS)
+        fts_field_names = map(lambda x: x['name'], self.FTS_FIELDS)
         dirty_fts_fields = set(fts_field_names).intersection(dirty_fields)
-        logging.debug("dirty_fts_fields: %r, current_doc=%r, current_ver=%r" % \
-                        (dirty_fts_fields,current_doc,current_doc[self.VERSION_FIELD_NAME] if current_doc is not None else "N/A"))
-        if len(dirty_fts_fields) > 0 or current_doc is None or current_doc[self.VERSION_FIELD_NAME] is None or current_doc[self.VERSION_FIELD_NAME]  < self.FTS_VERSION:
+
+        #logging.debug("dirty_fts_fields: %r, current_doc=%r, current_ver=%r" % \
+        #                (dirty_fts_fields,current_doc,current_doc[self.VERSION_FIELD_NAME] if current_doc is not None else "N/A"))
+
+        #if len(dirty_fts_fields) > 0 or current_doc is None or \
+        #    current_doc[self.VERSION_FIELD_NAME] is None or \
+        #    current_doc[self.VERSION_FIELD_NAME] < self.FTS_VERSION:
+
+        if 1:
```

Comment:
Another debug hack - it forces updating the existing documents when I change their structure.